internal/dispatch/caching/caching.go

```diff
@@ -405,10 +419,14 @@ func (cd *Dispatcher) DispatchLookupSubjects(req *v1.DispatchLookupSubjectsReque
 	if cachedResultRaw, found := cd.c.Get(requestKey); found {
 		cd.lookupSubjectsFromCacheCounter.Inc()
 		for _, slice := range cachedResultRaw.([][]byte) {
+			// slice, response := slice, v1.DispatchLookupSubjectsResponse{}
+			// response.Marshaled = slice // Stash the marshaled form for re-use
```

Comment:
I have this commented out for the moment until we figure out how to avoid a panic when 		`dispatchpkg.AddResponseMetadata(respMetadata, result.Metadata)` gets called.

Reply:
Any idea how I should go about this @josephschorr